City-of-Pori/pori-kada
# PORIJATE-16 Waste type page

To test:

1. drush fra -y && drush cc all
2. Go to edit the waste_type taxonomy (the name is now just Waste) and translate it to Finnish
3. Go to add waste type. Write a short description and full description for the node
4. Add some images
5. Go to view the taxonomy term page. Make sure there is header, footer and pretty slideshow and content.
6. Make sure you can translate the term and that the translated term page looks good too
7. Make sure that Finnish path is jate/[waste-name-in-finnish] and English is waste/[waste-name-in-english]
